constants.js

```diff
@@ -63,13 +63,13 @@ const dontMergeLabelCheck = 'dont-merge-label-check';
 const checksWhitelist = {
   // eslint-disable-next-line quote-props
   'oppia-android': {
-    [openEvent]: [claCheck],
+    [openEvent]: [claCheck, codeOwnerCheck],
```

Comment:
What does Oppiabot check for here? I'm mainly wondering how it will differ from the existing check already enabled in Oppia Android: https://github.com/oppia/oppia-android/blob/develop/.github/workflows/static_checks.yml#L19 (which uses https://github.com/mszostok/codeowners-validator).